images/scripts/rhel4edge.setup

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
This is almost the same as fedora-coreos.setup. Please rename that to `coreos.setup` and create two symlinks to it. Then please make the specifics (fedora or rhel, like zincati.service and installing extra packages)  dependent on `$0` or /etc/os-release.

Reply:
done, when I will create rhel4edge I will also refresh fedora-coreos to be sure